spec/gnip_rules_add_spec.rb

```diff
@@ -79,25 +79,37 @@
 
   context "location" do
     it 'hashtag and location' do
-      subject.hashtag('Foo').hashtag('bar').tag('footag').lat(123).lon(45).radius(0.01)
-      expect(subject.batch).to eq ([{"value"=>"#bar #foo point_radius:[123.0 45.0 0.01]", "tag"=>"footag"}])
+      subject.hashtag('Foo').hashtag('bar').tag('footag').lat(123).long(45).radius(0.01)
```

Comment:
Personally I don't feel too warm about this approach to setting values for an attribute of a class. I think value assignment should be done through an assignment operation (my_instance.long = value). I think passing the assignment value as an argument is an un-ruby way to do it.

I see that your `#long` method above is basically typecasting the argument to a float. You can accomplish the same thing by overriding the default assignment operator.

```
def long=(val)
  @long = val.to_f
end
```

Accomplishes the same thing as

```
def long(val)
  @long = val.to_f
end
```

With the benefit of being able to call `my_class.long = 123`. Inevitably this will change the way that your tests are structured if you choose that approach.